src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/CustomValidators/HtmlValidator.cs

```diff
@@ -14,24 +14,20 @@ internal class HtmlValidator : PropertyValidator
         internal HtmlValidator(IHtmlSanitizerService sanitizer) : base("{PropertyName} must contain allowed HTML")
         {
             _regex = CreateRegEx();
-
-
             _sanitizer = sanitizer;
         }
 
         protected override bool IsValid(PropertyValidatorContext context)
         {
             var value = (string) context.PropertyValue;
 
-            if (string.IsNullOrWhiteSpace(value))
-                return true;
-
             var isHtmlValid = _sanitizer.IsValid(value);
 
             if (isHtmlValid == false)
                 return false;
 
-            if (!_regex.IsMatch(value))
+            if (string.IsNullOrWhiteSpace(value) == false && 
```

Comment:
This is fine, just a concern on optional fields, would this fail validation on optional fields where html is allowed and the value is null?

Reply:
As it stands yes. This can be adapted later to handle optional fields when we have them.  We could pass in a bool when we attach the validator to manage this.